src/main/java/duke/task/TaskType.java

```diff
@@ -0,0 +1,46 @@
+package duke.task;
+
+/**
+ * A duke.task.TaskType represents a type of duke.task.Task in the application.
+ */
+public enum TaskType {
+    DEADLINE("D"),
+    EVENT("E"),
+    TODO("T");
+
+    /**
+     * The name of the duke.task.Task.
+     */
+    private final String name;
+
+    /**
+     * Creates a duke.task.TaskType with the specified name.
+     * @param name Name of the duke.task.TaskType.
+     */
+    TaskType(String name) {
+        this.name = name;
+    }
+
+    /**
+     * Gets the String representation of the duke.task.TaskType.
+     * @return The String representation of the duke.task.TaskType
+     */
+    @Override
+    public String toString() {
+        return this.name;
+    }
+
+    /**
+     * Parse the input String into an equivalent duke.task.TaskType.

```

Comment:
It's better if you use the word _Parses_ instead of _Parse_.

Reply:
Nice catch. Thanks.